Makefile

```diff
@@ -25,12 +25,8 @@ test-import: export-blends
 
 
 update-examples:
-	mkdir -p tests/reference_exports
```

Comment:
Any reason we want to change this part?

If you ever launch Godot from `test/godot_project`, there would be Godot cache files generated, for example `*.escn.import` files, a plain `cp -r` would copy those unwanted files to `reference_exports`. Therefore, I prefer we can keep the current version that only copies *.escn files.

Reply:
I mentioned this in the commit message :)

> Also fix make update-examples, as the existing script had some limitations:

> - It expected a certain level of nesting, and failed for
>   tests/test_scenes/material_cycle/material_spatial/ due to the extra
>   level of nesting
> - It only copied .escn files, but we actually need to make sure other
>   files (like .png textures) are produced
> - It would not clean up files left behind from prior runs. For example,
>   if an update failed to produce a .png, but we had it left from the
>   prior run, we wouldn't notice.
> 
> The rm/cp approach is simpler and solves these problems.

Fair point about the cache though, and maybe my last point doesn't matter since CI will catch it even if your local machine doesn't.

Reply:
Still, `make compare` [ignores .escn.import](https://github.com/godotengine/godot-blender-exporter/blob/0c52742d0bc9f220ebfcca9c6f84a1b60a7ea8c2/Makefile#L32) and `.gitignore` [ignores .import files](https://github.com/godotengine/godot-blender-exporter/blob/0c52742d0bc9f220ebfcca9c6f84a1b60a7ea8c2/.gitignore#L8).

The `cp` approach isn't perfect, but I think it has less limitations than the current `find` approach.

Reply:
If `cp` is too limited to copy files, maybe look into using `rsync`. It can be used in a pure local fashion too :slightly_smiling_face:

Reply:
@rcorre Sorry for missing the commit message. Then let's go with the `cp -r`. The cache file problem is actually easy to solve, just by adding them to gitignore. 

@Calinou Thanks! Really useful tools learned : )

Reply:
@rcorre there is a conflict. Could you do a rebase? and I will merge it. Really appreciate your work!

Reply:
Yeah, `rsync` could totally solve this too, though not everyone might have it.